Dockerfile

```diff
@@ -30,3 +30,4 @@ RUN npm ci && npm run deploy
 
 WORKDIR /root
 RUN mix do phx.digest, release
+RUN upload_assets.sh
```

Comment:
Does the Docker image have the right credentials to upload to S3, or do we need to add those as well?